src/pharmpy/model/statements.py

```diff
@@ -347,8 +347,30 @@ def move_dose(self, source, destination):
         mapping = {source: new_source, destination: new_dest}
         nx.relabel_nodes(self._g, mapping, copy=False)
 
+    def move_oral_dose(self, source, destination, single_dose=True):
```

Comment:
Yes, `move_dose` has been updated with and argument `admid` instead of adding a new method. Also added argument `replace` to determine of the moved dose should replace the dose of the destination compartment or not